ui-components/seatmap.md

```diff
@@ -68,7 +68,29 @@ Ex -
 sendMessageToIframe("addSeat", { id })
 ```
 
-**removeSeat** - This event is triggered to add a seat within seatmap from any interaction outside Iframe. This expects an object with id element as data along with the event. Id is part of seat object.
+This is the contract of Seat Object
+```javascript
+Object: Seat
+{
+	id
+	currency
+	seatNumber,
+	seatRow: row.attr("display"),
+	color: seat.attr("allotedSeatColor"),
+	seatCode: seat.attr("display").concat(row.attr("display")),
+	seatSection: section.attr("display"),
+	remaining: parseInt(seat.attr("remaining")),
+	maxAvailable: parseInt(seat.attr("max-available")),
+	description: seat.attr("description"),
+	price,
+	originalPrice,
+	discounted: originalPrice > price,
+}
+```
+
+If you are wondering how do you get these values. Most of these values are part of html itself in Iframe. If you have to fetch it you have to fetch it from html attributes of seat node.
```

Comment:
Do we not return the seat object on selecting the seat?
I don't know if it's even possible

Reply:
Yes we do but UI doesn't have to store it. They can simply use it from HTML whenever needed